src/commands/game_commands/play.ts

```diff
@@ -1153,6 +1181,28 @@ export default class PlayCommand implements BaseCommand {
             }
         }
 
+        if (
+            gameType === GameType.CLIP &&
+            (!guildPreference.gameOptions.guessTimeout ||
+                guildPreference.gameOptions.guessTimeout >
+                    GuessTimeoutCommand.TIMER_MAX_ACCEPTABLE_CLIP_BEFORE_RESET)
+        ) {
+            await guildPreference.setGuessTimeout(
+                GuessTimeoutCommand.TIMER_DEFAULT_VALUE,
```

Comment:
Why are we comparing with TIMER_MAX_ACCEPTABLE_CLIP_BEFORE_RESET but capping it to TIMER_DEFAULT_VALUE?